_includes/master/icmp.md

```diff
@@ -1,4 +1,4 @@
-| Field       | Description                 | Accepted Values   | Schema | Default    |
-|-------------|-----------------------------|-------------------|--------|------------|
-| type | Match on ICMP type. | Can be integer 1-255 | integer |
-| code | Match on ICMP code. | Can be integer 1-255 | integer |
\ No newline at end of file
+| Field  | Description         | Accepted Values      | Schema  | Default    |
+|--------|---------------------|----------------------|---------|------------|
+| type   | Match on ICMP type. | Can be integer 0-254 | integer |
+| code   | Match on ICMP code. | Can be integer 0-254 | integer |
```

Comment:
Code actually _can_ be 255.  It's only type that's forbidden from being 255 (due to a kernel limitation).

Reply:
All right, thanks! I have updated this PR....let me know if it looks good by giving your approval and then I can squash and merge it. Thanks @fasaxc !